torch/csrc/jit/serialization/import.h

```diff
@@ -37,6 +37,12 @@ TORCH_API Module import_ir_module(
     c10::optional<c10::Device> device,
     ExtraFilesMap& extra_files);
 
+TORCH_API Module import_ir_module(
+    std::shared_ptr<CompilationUnit> cu,
+    std::shared_ptr<caffe2::serialize::PyTorchStreamReader> reader,
+    c10::optional<at::Device> device,
+    std::string ts_id);
```

Comment:
nit: the comment about what this overload does should probably be here as well (you have it in import.cpp already).  Also, a comment on what ts_id represents would be nice, since it's not obvious from the name.